include/mbgl/ios/MGLMapView.h

```diff
@@ -82,6 +82,14 @@ IB_DESIGNABLE
 /** The button shown in the lower-right of the map which when pressed displays the map attribution information. */
 @property (nonatomic, readonly) UIButton *attributionButton;
 
+/** Query the visible map for features at a given point. 
+*
+*   @warning Beta API. Return value subject to change in subsequent releases.
```

Comment:
Let's document this, but I've renamed the API in https://github.com/mapbox/mapbox-gl-native/commit/0ea4f537bbb48a1babccc2702b3e16dce9379a0c to make room for our final, more-richly-typed API once style API lands.